plugin/storage/integration/integration_test.go

```diff
@@ -179,20 +173,21 @@ func (s *StorageIntegration) testGetTrace(t *testing.T) {
 	s.refresh(t)
 
 	var actual *model.Trace
-	for i := 0; i < iterations; i++ {
-		s.logger.Info(fmt.Sprintf(waitForBackendComment, i+1, iterations))
+	found := s.waitForCondition(t, func(t *testing.T) bool {
 		var err error
 		actual, err = s.SpanReader.GetTrace(expectedTraceID)
-		if err == nil && len(actual.Spans) == len(expected.Spans) {
-			break
+		if err != nil {
+			t.Log(err)
```

Comment:
why does this test get special treatment? the other tests aren't logging the errors

Reply:
maybe they all should, but this is specifically the one that was sporadically failing yesterday for Kafka, so I want to get the logging into it for the future.